static/app/components/replays/player/styles.tsx

```diff
@@ -18,7 +18,9 @@ export const baseReplayerCss = css`
   .replayer-wrapper > iframe {
     border: none;
     background: white;
+  }
 
+  &[data-inspectable='true'] .replayer-wrapper > iframe {
```

Comment:
intentional?

Reply:
yeah. i should add a story for this

I remember we allowed pointer events, but in the app today they're disabled for some reason, the styles might be intentionally or unintentionally messed up, i can't sort out which... but pointer-event is effectively `none` in all the places afaik.

I split this out on purpose in case we wanted to allow pointer-events again.